### DISCUSSION TITLE
GSoC: Reaching out to mentors of Project 8 ( Improve Generative AI workload performance on ARM devices with OpenVINO)

### DISCUSSION THREAD

**Discussion by aooxin** [openvinotoolkit/openvino#23214] [Google Summer of Code]
Dear GSOC Project Mentors,

I am writing to apply for the GSOC project "Improve Generative AI workload performance on ARM devices with OpenVINO." As a Master’s student in Control Science and Engineering at Beijing Jiaotong University, with a background in Computer Science, I have focused on deep learning acceleration on heterogeneous platforms. My experience with OpenCL and passion for machine learning position me as a strong candidate for this project.

**Project Choice and Implementation Plan:**  
My choice of this project stems from a desire to deepen my expertise in OpenCL programming and AI acceleration. I plan to dedicate 350 hours to analyze performance bottlenecks and implement optimizations in OpenVINO's runtime for ARM devices. My approach will be methodical, starting with a review of OpenVINO documentation and culminating in the development of performance enhancements.

**Conclusion:**  
I am excited about the opportunity to apply my skills to "Improve Generative AI workload performance on ARM devices with OpenVINO" and contribute to the OpenVINO project through GSOC. Thank you for considering my application.

Sincerely,
Ma Aoxin

**Comment by adrianboguszewski** [comment]
@alvoron @dmitry-gorokhov please look at it

**Comment by alvoron** [comment]
@aooxin thank you for submitting your proposal!
Do you have Apple device with M chip?

**Reply by aooxin** [reply]
thank you for your reply. i have macbook pro with m1 silicon.

**Comment by aooxin** [comment]
@alvoron @dmitry-gorokhov Hi, mentors. Could you please provide me with further guidance on this project? Thank you very much!

**Comment by alvoron** [comment]
@aooxin sure, we'll bring some details to this thread on the next week.

**Reply by aooxin** [reply]
Hello, I would like to know if the relevant code repository is already open, and in addition could I apply directly on GSOC tomorrow
